numpy/lib/histograms.py

```diff
@@ -562,7 +562,9 @@ def histogram_bin_edges(a, bins=10, range=None, weights=None):
     below, :math:`h` is the binwidth and :math:`n_h` is the number of
     bins. All estimators that compute bin counts are recast to bin width
     using the `ptp` of the data. The final bin count is obtained from
-    ``np.round(np.ceil(range / h))``.
+    ``np.round(np.ceil(range / h))``. The final bin width is often less 
+    than what is returned by the formulas below, in order to allow the 
```

Comment:
Suggestion: "The final bin width will therefore generally be less than what is returned by the estimators." "estimator" is a better term than formula, and more consistent. The part about entire range is covered by `ptp` two sentences back.

Reply:
Thanks for your comment, will adjust and repush